pulpcore/cli/file/acs.py

```diff
@@ -59,8 +59,8 @@ def path() -> None:
 @href_option
 @acs_lookup_option
 @path_option
-@pass_entity_context
-def add(acs_ctx: PulpFileACSContext, paths: Iterable[str]) -> None:
+@pass_acs_context
+def add(acs_ctx: PulpACSContext, paths: Iterable[str]) -> None:
     """Add path(s) to an existing ACS."""
     paths = set(paths)
```

Comment:
Do we want/need an assert isinstance(acs_ctx, PulpFileACSContext) here?

Reply:
If it is only using the baseclass interface, why should we? Maybe it's even a sign this should be a generic command.

(And mypy would tell us if we used the detailed api.)